tests/src/kinAccount.intg.test.ts

```diff
@@ -171,5 +170,14 @@ describe("KinAccount", async () => {
 
 	}, 60000);
 
+	test("Create account with an existing address, expect error", async () => {
+
+		const txBuilder = await sender.buildCreateAccount({
+			address: receiver.publicAddress,
+			startingBalance: 10,
+			fee: 100
+		});
+		expect((txBuilder._server as any).headers.get("USER-AGENT")).toEqual(Server.GLOBAL_HEADERS.get("USER-AGENT"));
```

Comment:
not really testing the behavior (just the assignment), also integ test is not needed for that anyway, I would do a unit test with nock, telling nock to expect to get the input headers when reaching the network.